python/test/Ice/blobject/RouterI.py

```diff
@@ -70,7 +70,7 @@ def done(self, future):
             self._future.set_exception(ex)
 
 
-class BlobjectAsyncI(Ice.BlobjectAsync):
+class BlobjectAsyncI(Ice.Blobject):
```

Comment:
It would be nice to refactor (simplify) this test and use asyncio for the async portion. For a follow-up PR?